### PR TITLE
Here's my digital signature.

### PR DESCRIPTION
**Pull request by ARitz-Cracker**

### CODE REVIEW

File: ARitz-Cracker [hunk 1]
```diff
@@ -0,0 +1,11 @@
+2018-10-21
```
Comment (peol) [inline ARitz-Cracker]: Hi @ARitz-Cracker — the PR looks good, thanks!

We also need your e-mail, the second part of signing the CLA before we can merge this: https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sending-the-email

Please let us know once that's done and we'll get you sorted :)

Thank you!

**Review by peol — CHANGES_REQUESTED** [review]
Hi @ARitz-Cracker — the PR looks good, thanks!

We also need your e-mail, the second part of signing the CLA before we can merge this: https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sending-the-email

Please let us know once that's done and we'll get you sorted :)

Thank you!

**Review by peol — CHANGES_REQUESTED** [review]
Hi @ARitz-Cracker — the PR looks good, thanks!

We also need your e-mail, the second part of signing the CLA before we can merge this: https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sending-the-email

Please let us know once that's done and we'll get you sorted :)

Thank you!

**Review by peol — CHANGES_REQUESTED** [review]
Hi @ARitz-Cracker — the PR looks good, thanks!

We also need your e-mail, the second part of signing the CLA before we can merge this: https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sending-the-email

Please let us know once that's done and we'll get you sorted :)

Thank you!

**Review by peol — CHANGES_REQUESTED** [review]
Hi @ARitz-Cracker — the PR looks good, thanks!

We also need your e-mail, the second part of signing the CLA before we can merge this: https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sending-the-email

Please let us know once that's done and we'll get you sorted :)

Thank you!

**Review by peol — CHANGES_REQUESTED** [review]
Hi @ARitz-Cracker — the PR looks good, thanks!

We also need your e-mail, the second part of signing the CLA before we can merge this: https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sending-the-email

Please let us know once that's done and we'll get you sorted :)

Thank you!

**Review by peol — CHANGES_REQUESTED** [review]
Hi @ARitz-Cracker — the PR looks good, thanks!

We also need your e-mail, the second part of signing the CLA before we can merge this: https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sending-the-email

Please let us know once that's done and we'll get you sorted :)

Thank you!

**Review by peol — CHANGES_REQUESTED** [review]
Hi @ARitz-Cracker — the PR looks good, thanks!

We also need your e-mail, the second part of signing the CLA before we can merge this: https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sending-the-email

Please let us know once that's done and we'll get you sorted :)

Thank you!

**Review by peol — CHANGES_REQUESTED** [review]
Hi @ARitz-Cracker — the PR looks good, thanks!

We also need your e-mail, the second part of signing the CLA before we can merge this: https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sending-the-email

Please let us know once that's done and we'll get you sorted :)

Thank you!

**Review by peol — CHANGES_REQUESTED** [review]
Hi @ARitz-Cracker — the PR looks good, thanks!

We also need your e-mail, the second part of signing the CLA before we can merge this: https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sending-the-email

Please let us know once that's done and we'll get you sorted :)

Thank you!

**Review by peol — CHANGES_REQUESTED** [review]
Hi @ARitz-Cracker — the PR looks good, thanks!

We also need your e-mail, the second part of signing the CLA before we can merge this: https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sending-the-email

Please let us know once that's done and we'll get you sorted :)

Thank you!

**Review by peol — CHANGES_REQUESTED** [review]
Hi @ARitz-Cracker — the PR looks good, thanks!

We also need your e-mail, the second part of signing the CLA before we can merge this: https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sending-the-email

Please let us know once that's done and we'll get you sorted :)

Thank you!

**Review by peol — CHANGES_REQUESTED** [review]
Hi @ARitz-Cracker — the PR looks good, thanks!

We also need your e-mail, the second part of signing the CLA before we can merge this: https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sending-the-email

Please let us know once that's done and we'll get you sorted :)

Thank you!

**Review by peol — CHANGES_REQUESTED** [review]
Hi @ARitz-Cracker — the PR looks good, thanks!

We also need your e-mail, the second part of signing the CLA before we can merge this: https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sending-the-email

Please let us know once that's done and we'll get you sorted :)

Thank you!

**Review by peol — CHANGES_REQUESTED** [review]
Hi @ARitz-Cracker — the PR looks good, thanks!

We also need your e-mail, the second part of signing the CLA before we can merge this: https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sending-the-email

Please let us know once that's done and we'll get you sorted :)

Thank you!

**Review by peol — CHANGES_REQUESTED** [review]
Hi @ARitz-Cracker — the PR looks good, thanks!

We also need your e-mail, the second part of signing the CLA before we can merge this: https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sending-the-email

Please let us know once that's done and we'll get you sorted :)

Thank you!

**Review by peol — CHANGES_REQUESTED** [review]
Hi @ARitz-Cracker — the PR looks good, thanks!

We also need your e-mail, the second part of signing the CLA before we can merge this: https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sending-the-email

Please let us know once that's done and we'll get you sorted :)

Thank you!

**Comment by peol** [comment]
Sorry about the spam, GitHub had some real issues this morning...

**Comment by ARitz-Cracker** [comment]
I've sent the e-mail shortly submitting this PR.

image

**Review by peol — APPROVED** [review]

**Comment by peol** [comment]
@ARitz-Cracker You're absolutely right! It had gone into the junk mail folder for some reason. All good here, thank you again!

**Pull request merged**